pkg/googlecloud/bigtable_wrapper.go

```diff
@@ -25,17 +24,17 @@ func (c *clusterInfoWrapper) ServerNodes() int32 {
 }
 
 // Make sure the wrapper complies with its interface.
-var _ interfaces.ClusterInfoWrapper = (*clusterInfoWrapper)(nil)
+var _ ClusterInfoWrapper = (*clusterInfoWrapper)(nil)
```

Comment:
Take a look at my previous comment about the return type and struct initialization

```suggestion
```